docs/modules/nifi/pages/usage_guide/extra-volumes.adoc

```diff
@@ -1,6 +1,6 @@
-= Adding External Files to the NiFi Servers
+= Adding external files to the NiFi servers
 
-Since Apache NiFi allows executing pretty much arbitrary workflows depending on which processors are used, it may become necessary to add external files to the pods.
+Since Apache NiFi allows executing pretty much arbitrary workflows depending on which processors are used, it may become necessary to add external files to the Pods.
```

Comment:
```suggestion
Since Apache NiFi allows executing arbitrary workflows depending on which processors are used, it may become necessary to add external files to the Pods.
```